### PR TITLE
Hotfix/sellingdatevalidator

### PR DESCRIPTION
**Pull request by jhagberg**

### CODE REVIEW

File: app/utils/database.py [hunk 1]
```diff
@@ -970,7 +970,7 @@ def can_edit(self, identifier):
         `identifier` will be parsed as:
 
         `^([a-zA-Z][0-9]+-[0-9]+)$`: individual
-        `^([a-zA-Z][0-9]+)$`: herd
+        `^(([G|M]X1)|[a-zA-Z][0-9]+)$`: herd
```
Comment (andkaha) [inline app/utils/database.py]: `[G|M]` should be `[GM]` or just `.`.

File: app/utils/database.py [hunk 2]
```diff
@@ -1005,7 +1005,7 @@ def can_edit(self, identifier):
             except DoesNotExist:
                 pass
 
-        elif re.match("^([a-zA-Z][0-9]+)$", identifier):
+        elif re.match("^(([G|M]X1)|[a-zA-Z][0-9]+)$", identifier):
```
Comment (andkaha) [inline app/utils/database.py]: `[G|M]` should be `[GM]` or just `.`.

**Review by andkaha — CHANGES_REQUESTED** [review]
Error in regular expressions.

File: app/utils/database.py (hunk 1 above)
Comment (jhagberg) [reply app/utils/database.py]: If we merge the first PR I can rebase on Develop

Comment (andkaha) [reply app/utils/database.py]: The first one is merged. Go ahead and rebase and merge this one too.

**Review by andkaha — APPROVED** [review]
The test is reasonable. If the syntax is correct, I approve.

**Review by HenrikeW — APPROVED** [review]
Looks like it's working!

**Pull request merged**